nmap.go

```diff
@@ -111,6 +116,42 @@ func (s *Scanner) Run() (*Run, error) {
 	}
 }
 
+// RunAsync runs nmap asynchronously and returns error.
+func (s *Scanner) RunAsync() error {
+	// Enable XML output
+	s.args = append(s.args, "-oX")
+
+	// Get XML output in stdout instead of writing it in a file
+	s.args = append(s.args, "-")
+	s.Cmd = exec.Command(s.binaryPath, s.args...)
+
+	stderr, err := s.Cmd.StderrPipe()
+	if err != nil {
+		return err
+	}
+
+	stdout, err := s.Cmd.StdoutPipe()
+	if err != nil {
+		return err
```

Comment:
return fmt.Errorf("unable to get standard output from asynchronous nmap run: %v", err)

Reply:
Resolved with latest commit